src/ansys/fluent/core/workflow.py

```diff
@@ -358,29 +358,28 @@ def python_name(self) -> str:
         if not self._python_name:
             try:
                 this_command = self._command()
-                _help_string = camel_to_snake_case(this_command.get_attr("helpString"))
-                _disp_text = this_command.get_attr("displayText")
                 # temp reuse helpString
-                self._python_name = _help_string
-                if (
-                    self._python_name
-                    in self._command_source._help_string_display_text_map
-                ):
-                    self._populate_duplicate_task_list()
-                else:
-                    self._command_source._help_string_display_text_map[
-                        self._python_name
-                    ] = _disp_text
-                self._command_source._help_string_command_id_map[self._python_name] = (
-                    this_command.command
-                )
-                self._command_source._help_string_display_id_map[self._python_name] = (
-                    _disp_text
+                self._python_name = camel_to_snake_case(
+                    this_command.get_attr("helpString")
                 )
+                self._cache_data(this_command)
             except Exception:
                 pass
         return self._python_name
 
+    def _cache_data(self, command):
+        _disp_text = command.get_attr("displayText")
+        if self._python_name in self._command_source._help_string_display_text_map:
```

Comment:
It looks like this map should be named according to `python_name` keys rather than `help_string` keys, because that's how it's being keyed here. E.g., `_python_name_to_display_text`.

Reply:
Will do this once the behavior is fixed. Thank you.